### PR TITLE
Update DG

### PR DESCRIPTION
**Pull request by ReganChoy**
Fix manual testing format

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -930,36 +930,100 @@ testers are expected to do more *exploratory* testing.
...
+   4. Test case (Missing Parameters): `met d/2024-04-11`, `met 3` or any command with missing parameters<br>
+      Expected: Similar to previous
+   
+   5. Test case (Repeated Parameters): `met 3 d/2024-04-11 d/2024-04-11` or any command with repeated parameter<br>
+      Expected: Similar to previous.
+   
+   6. Test case (Invalid Index): `met x d/2024-04-11` (where x is smaller or larger than the list size)<br>
+      Expected: Similar to previous
+   
+   7. Test Case (Future Date, Invalid Date): `met 3 d/2024-04-20`
```
Comment (solomonng2001) [inline docs/DeveloperGuide.md:953]: Should we be setting a much later date like in 2025? The bug testers might still be testing it after 20 April.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -930,36 +930,100 @@ testers are expected to do more *exploratory* testing.
...
-Desired Outcome: Creating an appointment with the 3rd client on 18 April 2024 2pm.<br>
-1. Correct Test Case: `schedule 3 d/2024-04-18 14:00`
-2. Invalid Test Case: `schedule 3 d/18-04-2024 14:00` wrong dateTime format
-3. Invalid Test Case: `schedule 3 d/2024-04-18` missing time
+
+1. Scheduling an appointment with a client while all clients are being shown
+   1. Prerequisites: List all clients using the `list` command. Multiple clients in the client list. Ensure the date chosen is in the future.
+     For the test cases, we assume that today is 13 April 2024 and the time is 2pm.
+
+   2. Test case: `schedule 1 d/2024-04-18 18:00`<br>
```
Comment (solomonng2001) [inline docs/DeveloperGuide.md:965]: Shall we set this future schedule much later in the future as well?

**Review by solomonng2001 — DISMISSED** [review]
LGTM

**Review by yorklim — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1002,36 +1002,100 @@ testers are expected to do more *exploratory* testing.
...
+   4. Test case (Missing Parameters): `met d/2024-04-11`, `met 3` or any command with missing parameters<br>
+      Expected: Similar to previous.
+   
+   5. Test case (Repeated Parameters): `met 3 d/2024-04-11 d/2024-04-11` or any command with repeated parameter<br>
+      Expected: Similar to previous.
+   
+   6. Test case (Invalid Index): `met x d/2024-04-11` (where x is smaller or larger than the list size)<br>
+      Expected: Similar to previous.
+   
+   7. Test Case (Future Date, Invalid Date): `met 3 d/2025-04-20`
```
Comment (solomonng2001) [inline docs/DeveloperGuide.md:1025]: Add breaks

**Pull request merged**